mappings/net/minecraft/client/option/OptionInstance.mapping

```diff
@@ -0,0 +1,203 @@
+		ARG 1 key
+		ARG 2 tooltip
+		ARG 3 textGetter
+		ARG 4 values
+		ARG 5 codec
+		ARG 6 defaultValue
+		ARG 7 updateCallback
+	METHOD <init> (Ljava/lang/String;Lnet/minecraft/unmapped/C_vbibxkxv$C_hcvcuvxo;Lnet/minecraft/unmapped/C_vbibxkxv$C_tunipivd;Lnet/minecraft/unmapped/C_vbibxkxv$C_wevbfulp;Ljava/lang/Object;Ljava/util/function/Consumer;)V
+		ARG 1 key
+		ARG 2 tooltip
+		ARG 3 textGetter
+		ARG 4 values
+		ARG 5 defaultValue
+		ARG 6 updateCallback
+	METHOD m_akdeyqph createButton (Lnet/minecraft/unmapped/C_lscptbhn;III)Lnet/minecraft/unmapped/C_kpvuxmkp;
+		ARG 1 options
+		ARG 2 x
+		ARG 3 y
+		ARG 4 width
+	METHOD m_bwrczfzy emptyTooltip ()Lnet/minecraft/unmapped/C_vbibxkxv$C_hcvcuvxo;
+	METHOD m_bytnpyal (Ljava/lang/Boolean;)V
+		ARG 0 value
+	METHOD m_eedodylk (Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_ayfeobid;)Lnet/minecraft/unmapped/C_vbibxkxv$C_msgaqezv;
+		ARG 1 client
+	METHOD m_gdmjalon set (Ljava/lang/Object;)V
+		ARG 1 value
+	METHOD m_ihkotpvm ofBoolean (Ljava/lang/String;ZLjava/util/function/Consumer;)Lnet/minecraft/unmapped/C_vbibxkxv;
+		ARG 0 key
+		ARG 1 defaultValue
+		ARG 2 updateCallback
+	METHOD m_jivynsvq getCodec ()Lcom/mojang/serialization/Codec;
+	METHOD m_limrmpqe (Lnet/minecraft/unmapped/C_vbibxkxv$C_tunipivd;Ljava/lang/Object;)Lnet/minecraft/unmapped/C_rdaqiwdt;
+		ARG 2 value
+	METHOD m_lotxbajv getValues ()Lnet/minecraft/unmapped/C_vbibxkxv$C_wevbfulp;
+	METHOD m_minngzkm wrapTooltipLines (Lnet/minecraft/unmapped/C_ayfeobid;Lnet/minecraft/unmapped/C_rdaqiwdt;)Ljava/util/List;
+		ARG 0 client
+		ARG 1 tooltip
+	METHOD m_mtzljdir (Lnet/minecraft/unmapped/C_rdaqiwdt;Ljava/lang/Boolean;)Lnet/minecraft/unmapped/C_rdaqiwdt;
+		ARG 0 optionText
+		ARG 1 value
+	METHOD m_mwyoixie (Ljava/util/List;Ljava/lang/Object;)Ljava/util/List;
+		ARG 1 optionValue
+	METHOD m_nthzvuhj ofBoolean (Ljava/lang/String;Lnet/minecraft/unmapped/C_vbibxkxv$C_hcvcuvxo;Z)Lnet/minecraft/unmapped/C_vbibxkxv;
+		ARG 0 key
+		ARG 1 tooltip
+		ARG 2 defaultValue
+	METHOD m_nyawdjib ofBoolean (Ljava/lang/String;Lnet/minecraft/unmapped/C_vbibxkxv$C_hcvcuvxo;ZLjava/util/function/Consumer;)Lnet/minecraft/unmapped/C_vbibxkxv;
+		ARG 0 key
+		ARG 1 tooltip
+		ARG 2 defaultValue
+		ARG 3 updateCallback
+	METHOD m_ocjknhzx optionEnumText ()Lnet/minecraft/unmapped/C_vbibxkxv$C_tunipivd;
+	METHOD m_pgkhirfp (Lnet/minecraft/unmapped/C_rdaqiwdt;Lnet/minecraft/unmapped/C_okavjgbo;)Lnet/minecraft/unmapped/C_rdaqiwdt;
+		ARG 0 optionText
+		ARG 1 value
+	METHOD m_qbzovuqo (Ljava/lang/Object;)Ljava/util/List;
+		ARG 0 optionValue
+	METHOD m_rjfulvkg constantTooltip (Lnet/minecraft/unmapped/C_rdaqiwdt;)Lnet/minecraft/unmapped/C_vbibxkxv$C_hcvcuvxo;
+		ARG 0 text
+	METHOD m_snmuxvjl getValue ()Ljava/lang/Object;
```

Comment:
```suggestion
	METHOD m_snmuxvjl get ()Ljava/lang/Object;
		COMMENT {@return the value of this option}
```

To fix the inconsistency between set and get, the fact it's the value is already pretty obvious.